packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -109,6 +110,22 @@ const AccountSwitcher = observer(({ history, is_mobile, is_visible }) => {
         }
     };
 
+    const { OAuth2Logout } = useOAuth2(logoutRedirect);
+
+    const handleLogout = async () => {
+        closeAccountsDialog();
+        if (is_positions_drawer_on) {
+            togglePositionsDrawer(); // TODO: hide drawer inside logout, once it is a mobx action
+        }
+
+        if (is_oauth2_hydra_enabled) {
```

Comment:
[R] this value is not boolean, you may checkout the p2p implementation on how we extract the value from hydra_be to determine oauth enabled

Reply:
Yeap still in progress this one. I need to go in for the enabled_for